fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/CacheWeightCheckRequestHandler.java

```diff
@@ -148,17 +144,14 @@ protected SuccessCallback buildCacheSuccessCallback(
             DruidAggregationQuery druidQuery,
             ResponseProcessor response
     ) {
-        return new SuccessCallback() {
-            @Override
-            public void invoke(JsonNode jsonResult) {
-                // send the response to the user before waiting to cache
-                successCallback.invoke(jsonResult);
-                try {
-                    querySignedCacheService.writeCache(response, jsonResult, druidQuery);
-                } catch (JsonProcessingException e) {
-                    // Warn on cache write exception only
-                    LOG.warn("Cache write json exception:", e);
-                }
+        return jsonResult -> {
+            // send the response to the user before waiting to cache
+            successCallback.invoke(jsonResult);
+            try {
+                querySignedCacheService.writeCache(response, jsonResult, druidQuery);
+            } catch (JsonProcessingException e) {
+                // Warn on cache write exception only
+                LOG.warn(String.format("Caching writing exception. %s", e.getMessage()), e);
```

Comment:
Is this better than using the LOG formatting?
```suggestion
                LOG.warn("Caching writing exception. {}", e.getMessage(), e);
```

Reply:
I wanted to take advantage of the logger option that expects throwable. Those two syntaxes (syntactes?) (variable substitution and throwable) are incompatible.

Reply:
and the whole point of this block that we're fixing is to capture the trace on issues.